Sources/Optimizely/OptimizelyClient.swift

```diff
@@ -259,7 +259,7 @@ open class OptimizelyClient: NSObject {
                             variation: variation,
                             userId: userId,
                             attributes: attributes,
-                            flagKey: experimentKey,
+                            flagKey: "",
                             ruleType: "experiment")
```

Comment:
Do we have this new ruleType "experiment"? If so, can we add it to DecisionSource enum?